test/invalid/baseline/const-nullable.txt

```diff
@@ -0,0 +1,3 @@
+Syntax error at line 3, since `interface MyConstants`:
+  const boolean? ARE_WE_THERE_YET = false;
+               ^ No name for const
```

Comment:
I wonder if we should identify this case... though I can't imagine anyone trying to do this.